aspnetcore/tutorials/first-mvc-app/new-field.md

```diff
@@ -63,7 +63,7 @@ Update the `SeedData` class so that it provides a value for the new column. A sa
 [!code-csharp[Main](./start-mvc/sample2/src/MvcMovie/Models/SeedDataRating.cs?highlight=6&range=25-32)]
 
 >[!WARNING]
-> You must stop IIS Express before you run the `dotnet ef` commands.   See [To Stop IIS Express:](adding-model.md#stop-iis-express-reference-label)
+> You must stop IIS Express before you run the `dotnet ef` commands. In Visual Studio 2017 RC this step isn't necessary, but you should close the browser window and stop debugging. See [To Stop IIS Express:](adding-model.md#stop-iis-express-reference-label)
```

Comment:
No need to close browser window, in fact we encourage folks to keep it open. If you're Debugging, you'll get quickly trained to do it right. The general work flow is make code changes and then refresh the browser (not in debug mode)